src/netex-convertor/types.ts

```diff
@@ -36,14 +36,26 @@ export interface FareZone {
     prices: FareZonePrices[];
 }
 
```

Comment:
We will also need to update the rest of the netex-output-service to use the new MatchingSingleData rather than MatchingData

Reply:
Matching has an or to use either export type MatchingData = MatchingSingleData | MatchingReturnData;